aiohttp_client_cache/backends/mongodb.py

```diff
@@ -71,7 +71,7 @@ async def bulk_delete(self, keys: set):
     async def delete(self, key: str):
         spec = {'_id': key}
         if hasattr(self.collection, "find_one_and_delete"):
```

Comment:
Is this something not available in very old `motor` (`pymongo`) version? I did not notice this in a changelog. 

I will highly appreciate a comment about the purpose of this `hasattr(self.collection, "find_one_and_delete")` check.

Reply:
> Is this something not available in very old `motor` (`pymongo`) version? 

Good question! Short version: yes, and this check can definitely be removed now.

Longer version:
After doing a bit of code archaeology, it looks like this was a carryover from [this commit](https://github.com/requests-cache/requests-cache/commit/a56d45bacf96ca96ff5c6ab5c3170330a1bbf661) in requests-cache. `find_one_and_delete()` was added to pymongo in v3.0 (in [this issue](https://jira.mongodb.org/browse/PYTHON-821) and [this PR](https://github.com/mongodb/mongo-ruby-driver/pull/646)). So, this check would have been for compatibility with pymongo < 3.0, since at the time (August 2016) pymongo 3 was only one year old. The minimum version we're supporting in this library is motor 3 / pymongo 4, so we're safe to remove this.